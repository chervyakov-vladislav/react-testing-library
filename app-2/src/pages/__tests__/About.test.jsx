import { screen, render } from '@testing-library/react';

import { About } from '../About';

describe('About', () => { 
  it('рендер About', () => {
    render(<About />);

    expect(screen.getByRole('heading')).toMatchSnapshot();
  });
});