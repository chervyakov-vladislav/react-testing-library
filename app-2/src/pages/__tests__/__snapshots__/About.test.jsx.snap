// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About рендер About 1`] = `
<h1>
  Hello, from About page
</h1>
`;
